packages/mermaid/src/dagre-wrapper/markers.js

```diff
@@ -11,6 +48,8 @@ const insertMarkers = (elem, markerArray, type, id) => {
 
 const extension = (elem, type, id) => {
   log.trace('Making markers for ', id);
+  let backgroundColor = getBackgroundColor(elem);
```

Comment:
```suggestion
  const backgroundColor = getBackgroundColor(elem);
```

This should be `const` if we never change the `backgroundColor` variable.

Reply:
you are correct - originally it would change, but I did not adjust when implementation was improved

Reply:
> you are correct - originally it would change

We should probably add some sort of ESlint rule that automatically catches this and fixes this for us.